docs/sources/operations/caching.md

```diff
@@ -31,7 +31,7 @@ To enable and configure Memcached:
        ```
        --memory-limit=4096 --max-item-size=2m --conn-limit=1024
        ```
-    1. Query result cache
+    1. Query result, index queries and writes cache
```

Comment:
The instructions are a big vague here (and I know this is existing docs) and don't clearly specify where to make these configuration edits.  If this is three separate configuration files that need to be modified, we should list it as THREE separate steps, to make it absolutely clear to users that there are three separate places that this configuration needs to be applied.

Reply:
Good point. The edits are in three sections of the same configuration file. I've tried to clarify this.